tests/integration/transactions/test_trust_set.py

```diff
@@ -92,3 +93,45 @@ async def test_special_chars_curr_codes(self, client):
             error.exception.args[0],
             "{'currency': 'Invalid currency abcd'}",
         )
+
+    @test_async_and_sync(globals())
+    async def test_deep_freeze_functionality(self, client):
+        issuer_wallet = Wallet.create()
+        response = await sign_and_reliable_submission_async(
+            TrustSet(
+                account=WALLET.address,
+                flags=TrustSetFlag.TF_SET_FREEZE & TrustSetFlag.TF_SET_DEEP_FREEZE,
+                limit_amount=IssuedCurrencyAmount(
+                    issuer=issuer_wallet.address,
+                    currency="USD",
+                    value="100",
+                ),
+            ),
+            WALLET,
+            client,
+        )
+
+        self.assertTrue(response.is_successful())
+        self.assertEqual(response.result["engine_result"], "tesSUCCESS")
+
+        account_lines_response = await client.request(
+            AccountLines(
+                account=WALLET.address,
+            )
+        )
+
+        self.assertTrue(account_lines_response.result.lines[0].freeze)
+
+        account_objects_response = await client.request(
+            AccountObjects(
+                account=WALLET.address,
+            )
+        )
+
+        self.assertTrue(
+            (
+                account_objects_response.result.account_obj[0]
+                | TrustSetFlag.TF_SET_DEEP_FREEZE
```

Comment:
@achowdhry-ripple I still don't see the bitwise AND operator here. Have you pushed all of your recent commits upstream ?